lib/chef/knife/azure_server_create.rb

```diff
@@ -245,6 +245,14 @@ def load_winrm_deps
       :default => false,
       :description => "Set this flag to enable auto chef client update in azure chef extension. This flag should be used with cloud-api bootstrap protocol only"
 
+      option :winrm_max_timeout,
+        :long => "--winrm-max-timeout MILISECONDS",
```

Comment:
Actually, we should use minutes to be consistent with other time options in knife-azure. Also, minutes is just more usable -- it's much easier to specify minutes than it is to convert your desired minutes (in this case most likely thought of by the user in minutes) to milliseconds.